jax/_src/scipy/linalg.py

```diff
@@ -1041,3 +1041,15 @@ def toeplitz(c: ArrayLike, r: Optional[ArrayLike] = None) -> Array:
       (nrows,), (1,), 'VALID', dimension_numbers=('NTC', 'IOT', 'NTC'),
       precision=lax.Precision.HIGHEST)[0]
   return jnp.flip(patches, axis=0)
+
+
+@_wraps(scipy.linalg.null_space)
+def null_space(A: ArrayLike, rcond: Optional[float] = None) -> Array:
```

Comment:
One thing we could do here is add an optional keyword-only `size` argument (similar to what we use in [`jnp.nonzero`](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.nonzero.html) and other APIs) that allows the user to specify a static size if they want the operation to be JIT-compatible. What do you think?

Reply:
Just saw what was done in `jnp.nonzero`, will do the same